csv-bulk-signature/src/main.ts

```diff
@@ -92,8 +95,13 @@ async function generateSignatures(
     );
   });
 
-  console.log("Number of signatures processed", processedRows.length);
-  console.log("Number of signatures skipped", skippedRows.length);
+  console.log(
+    `${skippedRows.length ? "🙅‍♂️" : "👍"} Signatures:`,
+    processedRows.length,
+    `processed and`,
+    skippedRows.length,
+    `skipped`
+  );
```

Comment:
- add emoji to front to show status if any were skipped
- keep number color/style in terminal by exposing a number instead of a string literal for the whole message